deploy/overlays/openshift/kustomization.yaml

```diff
@@ -8,11 +8,3 @@ bases:
 
 patchesStrategicMerge:
   - deployment.yaml
```

Comment:
The only difference between the OpenShift deployment and the standard one is that we enable selinux. @jhrozek do we wanna push this logic into the manager as well?

Reply:
I think it would be nice, to enable SELinux by default through the manager as well.

Reply:
So maybe if it’s not set explicitly in the config (not sure how to determine that), then we enable it automatically on OpenShift.